### ISSUE TITLE
Canu crashes after beginning assembly with message "Overlap error adjustment jobs failed, tried 2 times, giving up"

### ISSUE THREAD

**Issue by zehanna** [marbl/canu#2051]
Hello,

I'm trying to run Canu (v2.2) on our Slurm server for my PacBio Hifi reads using the command:

```
canu -assemble -p asm -d asm genomeSize=100m -pacbio-hifi reads.fastq.gz
```

It crashes about 2 hours after starting with this message in the canu.out file:

```
-- BEGIN ASSEMBLY
--
-- Overlap error adjustment jobs failed, tried 2 times, giving up.
--   job 00001.oea FAILED.
--   job 00002.oea FAILED.
--   job 00003.oea FAILED.
--   job 00004.oea FAILED.
--   job 00005.oea FAILED.
--   job 00006.oea FAILED.
--   job 00007.oea FAILED.
--   job 00008.oea FAILED.
--   job 00009.oea FAILED.
--   job 00010.oea FAILED.
--   job 00011.oea FAILED.
--   job 00012.oea FAILED.
--   job 00013.oea FAILED.
--   job 00014.oea FAILED.
--   job 00015.oea FAILED.
--   job 00016.oea FAILED.
--   job 00017.oea FAILED.
--   job 00018.oea FAILED.
--   job 00019.oea FAILED.
--   job 00020.oea FAILED.
--   job 00021.oea FAILED.
--   job 00022.oea FAILED.
--   job 00023.oea FAILED.
--

ABORT:
ABORT: canu 2.2
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
```
I have restarted the script already with the same outcome.
I don't really know what the problem is unfortunately, could you be so kind to help me find out?

Thanks a lot in advance,

Hanna

**Comment by skoren** [comment]
Post the log from one of the failing jobs, something like `unitigging/3-*/oea.*.out`

**Comment by zehanna** [comment]
Hi, I found out that the issue stemmed from the grid array jobs running out of time (which be default on our server is set to 1 hour).
With the option gridOptions="--time=10:00:00" I could extend the set time limit, however I think it was still not enough for some of the grid jobs since it aborted again. Therefore, I used canu without grid by defining useGrid=false. Like this, it ran without problems. Thanks a lot :)
